CommonTools/TriggerUtils/src/GenericTriggerEventFlag.cc

```diff
@@ -137,10 +137,16 @@ void GenericTriggerEventFlag::initRun( const edm::Run & run, const edm::EventSet
     if ( onL1_ && l1DBKey_.size() > 0 ) {
       const std::vector< std::string > exprs( expressionsFromDB( l1DBKey_, setup ) );
       if ( exprs.empty() || exprs.at( 0 ) != configError_ ) l1LogicalExpressionsCache_ = exprs;
+      for ( unsigned iExpr = 0; iExpr < l1LogicalExpressionsCache_.size(); ++iExpr ) {
+	std::string l1LogicalExpression( l1LogicalExpressionsCache_.at( iExpr ) );  
```

Comment:
I'm missing the point of these lines: it's a loop that assigns a value to a temporary that's never used.